virt/virtual_machines/virtual_disks/virt-expanding-vm-disk.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+[id="virt-vm-disk-resizing"]
+= Expanding a virtual machine disk
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-expanding-vm-disk
+
+toc::[]
+
+You can enlarge the size of a virtual machine's (VM) disk to provide a greater storage capacity by resizing the disk's persistent volume claim (PVC).
+
+However, you cannot reduce the size of a VM disk.
+
+include::modules/virt-enlarging-vm-disk.adoc[leveloffset=+1]
+
+[id="additional-resources_virt-vm-disk-resizing"]
+== Additional resources
+
+* link:https://docs.microsoft.com/en-us/windows-server/storage/disk-management/extend-a-basic-volume[Extend a basic volume in Windows].
+* link:https://access.redhat.com/solutions/29095[Extending an existing file-system partition without destroying data].
```

Comment:
Add "in Red Hat Enterprise Linux" at the end the link text here.

`Extending an existing file system partition without destroying data in Red Hat Enterprise Linux`